python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -736,7 +736,8 @@ def _reduce(
             Computed {op} of values within each group.
 
         .. pandas-compat::
-            **{cls}.{op}**
+            :meth:`pandas.core.groupby.DataFrameGroupBy.{op}`,
+             :meth:`pandas.core.groupby.SeriesGroupBy.{op}
```

Comment:
Assuming these missed backticks are a typo?

```suggestion
             :meth:`pandas.core.groupby.SeriesGroupBy.{op}`
```

Also applies to the updates to `GroupBy.apply` and `shift` below

Reply:
I think so. I'll take a look. Thanks

Reply:
should be fixed now